### DISCUSSION TITLE
[Config Support]: Always display camera name

### DISCUSSION THREAD

**Discussion by zinahe** [blakeblackshear/frigate#13372] [Config Support]
### Describe the problem you are having

Hi,

First of all, I would like to thank you the developers for this amazing tool. 

I am wondering if there is a way to configure Frigate to always display the camera name in the Live view. I understand it is shown whenever the connection is lost to a camera, but it kind of disappears as soon as the it becomes live.

Thanks.

### Version

1.9

### Frigate config file

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

CPU (no coral)

**Comment by hawkeye217** [comment]
No, there is no way to do this in the 0.14 UI. 

If your camera’s firmware supports it, you could overlay text on your stream there.

**Reply by zinahe** [reply]
@hawkeye217,

Thank you for the prompt reply. I am feeding an H.264 RTSP stream in to Frigate. And as such, it would be a waste of computing resources to re-encode the frames just for this. I wouldn't mind if the name is displayed outside of the camera frame (as part of the Frigate UI), but then that would mean an enhancement request.

Cheers,

**Reply by wolflu05** [reply]
I'd like to have this too. With a lot of cameras, labels would be really helpful. @zinahe Have you filled a feature request yet?

**Reply by NickM-27** [reply]
Adding the name doesn't require re encoding, most cameras have an option for it.

**Reply by zinahe** [reply]
@NickM-27, as you said it correctly "most" cameras could do it. Not all.

**Reply by wolflu05** [reply]
It would be really nice if there would be an option at: ui.show_camera_labels or even per camera to add an extra label in the top right corner above the camera preview with the camera name, like the label which is shown when no signal is received from the camera.

**Reply by bigthrilla** [reply]
Agreed, I would like to see the camera label outside of the stream.  I have it in the stream, but it's not clearly legible.
